editor/src/uuiui/radix-components.tsx

```diff
@@ -293,7 +293,7 @@ export const RadixSelect = React.memo(
               borderRadius: 6,
               color: 'white',
               boxShadow: UtopiaStyles.shadowStyles.high.boxShadow,
-              zIndex: 3,
+              zIndex: 'auto',
```

Comment:
this is the initial value, are you setting this here in case the className would override it to something non-initial?

Reply:
so it needs an explicit z-index value, otherwise it is hidden behind the modal, even though their portals are in the correct order in the DOM. however simply 'auto' didn't fix it as well, so I'm passing it from outside now.